index.html

```diff
@@ -17,7 +18,10 @@
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
   <link rel="stylesheet" href="./CSS/style.css" />
 </head>
+
```

Comment:
- [ ] Please re-write the title tag as the requirement say  see the screenshot 👇

![kkk](https://user-images.githubusercontent.com/61562626/130761856-609ce6b7-353a-4fc3-827f-9ca3cbf5424d.jpeg)

Reply:
The Pull request Titles?